### PR TITLE
questdb: init at 7.1.3

### PR DESCRIPTION
**Pull request by jakub-pravda**
###### Description of changes

QuestDB is an open source time-series database. The database is column-oriented and optimized for high-speed ingest and blazingly fast SQL analytics.

https://questdb.io/

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -40630,4 +40630,6 @@ with pkgs;
   isolate = callPackage ../tools/security/isolate { };
 
   reindeer = callPackage ../development/tools/reindeer { };
+
+  questdb = callPackage ../servers/nosql/questdb { };
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:40634]: Please sort this into a fitting category and sort there alphabetically.

File: pkgs/servers/nosql/questdb/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  pname = "questdb";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/questdb/questdb/releases/download/7.1.3/questdb-7.1.3-no-jre-bin.tar.gz";
+    sha256 = "941de1f07450690c1d4edc718c735f62b0d76375142d24ab33be0e08680ba0c7";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ jre ];
```
Comment (SuperSandro2000) [inline pkgs/servers/nosql/questdb/default.nix:17]: This is unused if it the only user is installPhase below

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (jakub-pravda) [reply pkgs/top-level/all-packages.nix:40634]: I have moved it, hopefully to the right place (but not sure, the file is pretty large and little bit messy)

File: pkgs/servers/nosql/questdb/default.nix [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp questdb.sh $out/bin
+    cp questdb.jar $out/bin
```
[the remark this answers is not in the document]
Comment (jakub-pravda) [reply pkgs/servers/nosql/questdb/default.nix]: Makes sense, done

**Comment by jakub-pravda** [comment]
I have made requested changes, can you take another look @SuperSandro2000, please?

**Comment by SuperSandro2000** [comment]
I get a notification when you force push but due to the amount of notifications I receive the usual wait time is a few days.

**Comment by jakub-pravda** [comment]
> I get a notification when you force push but due to the amount of notifications I receive the usual wait time is a few days.

No worries, thanks for the review

**Pull request merged**